core/trino-main/src/main/java/io/trino/execution/scheduler/PipelinedStageExecution.java

```diff
@@ -593,6 +593,11 @@ public boolean isFailure()
             return failureState;
         }
 
+        public boolean isRunningOrDone()
```

Comment:
Can we have tests for this?

Reply:
Is that necessary? It's the same as `SqlQueryScheduler.DistributedStagesSchedulerState.isRunningOrDone`.